debian/copyright

```diff
@@ -0,0 +1,21 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: k3s
+Upstream-Contact: Derek Nola <derek.nola@suse.com>
+
+Files: *
+Copyright: 2021 K3s Project Authors
```

Comment:
still valid?

Reply:
reading the docs I don't thing I should remove it. Kept the upstream then added our own for debian/*